FWCore/Framework/interface/EDProducer.h

```diff
@@ -62,21 +62,21 @@ namespace edm {
   private:
     bool doEvent(EventPrincipal const& ep, EventSetup const& c, ActivityRegistry* act, ModuleCallingContext const* mcc);
     // Needed by WorkerT but not supported
-    void preActionBeforeRunEventAsync(WaitingTask* iTask,
+    void preActionBeforeRunEventAsync(WaitingTask*                iTask,
                                       ModuleCallingContext const& iModuleCallingContext,
-                                      Principal const& iPrincipal) const {}
+                                      Principal const&            iPrincipal) const {}
 
     void doPreallocate(PreallocationConfiguration const&) {}
     void doBeginJob();
     void doEndJob();
     void doBeginRun(RunPrincipal const& rp, EventSetup const& c, ModuleCallingContext const* mcc);
     void doEndRun(RunPrincipal const& rp, EventSetup const& c, ModuleCallingContext const* mcc);
     void doBeginLuminosityBlock(LuminosityBlockPrincipal const& lbp,
-                                EventSetup const& c,
-                                ModuleCallingContext const* mcc);
+                                EventSetup const&               c,
+                                ModuleCallingContext const*     mcc);
```

Comment:
Just curious, is there a way to get the `const`s aligned as well?

Reply:
I don't see one